ledger/ledgercore/accountresource.go

```diff
@@ -30,3 +30,25 @@ type AccountResource struct {
 	AppLocalState *basics.AppLocalState
 	AppParams     *basics.AppParams
 }
+
+// ToAccountResource returns ledgercore.AccountResource for a creatable in basics.AccountData
```

Comment:
I'd suggest moving this method into the evalindexer; This method is only temporary ( until the indexer will get refactored to take advantage of the new state deltas ). Having the method here would lead to forgetting to delete it once the evalindexer is updated. ( note that there is nothing wrong with the implementation )

Reply:
There are linters ("unused", "deadcode") linters that I have been meaning to enable for that ... but will move